src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -1525,7 +1524,8 @@ FixedwingPositionControl::control_auto_takeoff(const hrt_abstime &now, const flo
 					   _param_fw_thr_min.get(),
 					   _param_fw_thr_max.get(),
 					   _param_sinkrate_target.get(),
-					   _performance_model.getMaximumClimbRate(_air_density));
+					   _performance_model.getMaximumClimbRate(_air_density),
+					   true); // disable underspeed handling
```

Comment:
```suggestion
					   static constexpr bool disable_underspeed_handling = true;
					   _performance_model.getMaximumClimbRate(_air_density),
					   disable_underspeed_handling);
```

such that we never end up with `false); // disable underspeed handling` in the future

Reply:
done in 4d7b014871b8c5e0fb94504256629b0a48e2ec20